src/renderer/components/widgets/channels/ListingMessage.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react'
-import PropTypes from 'prop-types'
-import * as R from 'ramda'
 
 import Grid from '@material-ui/core/Grid'
-import { withStyles } from '@material-ui/core/styles'
+import { makeStyles } from '@material-ui/core/styles'
 import { Typography } from '@material-ui/core'
 
 import BasicMessage from '../../../containers/widgets/channels/BasicMessage'
 import { _DisplayableMessage } from '../../../zbay/messages'
+import { DisplayableMessage } from '../../../zbay/messages.types'
 
 const reqSvgs =
+  // eslint-disable-next-line
   require && require.context('../../ui/assets/backgrounds', true, /\.svg$/)
```

Comment:
this trick will not work in typescript

But I don't think we need this component at all. Remove this